packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -154,7 +154,7 @@ class Auth {
   final http.Client _httpClient;
   final String _credentialsDir;
   final ObtainAccessCredentials _obtainAccessCredentials;
-  final CodePushClientBuilder _buildCodePushClient;
+  final CodePushClientBuilder _codePushClientWrapper;
```

Comment:
```suggestion
  final CodePushClientWrapper _codePushClientWrapper;
```

Also we can use the scoped instance instead of injecting it manually.